g2p/convert.py

```diff
@@ -16,14 +16,16 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    def __init__(self, bliss_lexicon):
+    def __init__(self, bliss_lexicon, all_phons=False):
```

Comment:
This will break existing hashes. What you can do is add:
```python
__sis_hash_exclude = {'all_phons': False}
```
above the init.

But I think the parameter name is not really nice. I would do it the other way round, and call it `write_first_phoneme_only` or so with "legacy" default True.